Source/Core/DolphinQt/Config/LogWidget.h

```diff
@@ -5,6 +5,7 @@
 #pragma once
 
 #include <QDockWidget>
+#include <QPlainTextEdit>
```

Comment:
This can be replaced with a forward declaration (the existing forward declarations can also be trimmed of the types you removed as well).